site/sfguides/src/data_mapping_in_native_apps/data_mapping_in_native_apps.md

```diff
@@ -60,27 +61,101 @@ The application provides a user interface which allows the consumer of the appli
 
 The first solution is not really what we want to be doing because the consumer will potentially have to create objects on their Snowflake instance just to satisfy the application's requirements, so this Quickstart will deliver the second solution.
 
+> aside positive
+> 
+> **Note** - The following steps explain how to create the app using the Snowflake CLI, as this option presents as a faster, more straightforward way to build Native Apps. You can also accomplish it by manually creating the folder structure and executing the SQL commands in Snowsight, but this alternative is out of the scope for this Quickstart.
+
+<!-- ------------------------ -->
+## Project Structure
+Duration: 7
+
+The project we are about to create is composed of several files in the following tree structure:
+
+```
+-- Datamapping App project
+
+    |-- app
+    |   |-- setup_script.sql
+    |   |-- manifest.yml
+    |   |-- ui
+    |   |   |-- enricher_dash.py
+    |-- scripts
+    |   |-- setup-package-script.sql
+    |-- snowflake.yml
+    |-- prepare_data.sh
+    
+```
+
+To start creating this folder structure, go to your console and inside the folder you want the project to reside. Execute the following **`snow app init datamapping_app_streamlit`**. This will create several files for a basic template, we are replacing some of them and also creating new ones.
```

Comment:
```suggestion
To start creating this folder structure, go to your console and inside the folder you want the project to reside. Execute the following command: **`snow app init datamapping_app_streamlit`**. 

This will create several files from our basic template; we will replacing some of them, as well as create some new ones.
```